ui/src/pages/frontsite/Documents/DocumentsDetails/LoanRequestForm/index.js

```diff
@@ -2,12 +2,18 @@ import { connect } from 'react-redux';
 import LoanRequestFormComponent from './LoanRequestForm';
 import { requestLoanForDocument } from './state/actions';
 
+const mapStateToProps = state => ({
+  isLoading: state.loanRequestForm.isLoading,
```

Comment:
it would be nice to add `data` here to that after request success you can display "You loan request is bla bla bla...". Or maybe in another PR?